timeseries/setup.py

```diff
@@ -32,10 +32,8 @@
     "accelerate",  # version range defined in `core/_setup_utils.py`
     "gluonts>=0.14.0,<0.14.4",  # 0.14.4 caps pandas<2.2
     "networkx",  # version range defined in `core/_setup_utils.py`
-    # TODO: update statsforecast to v1.5.0 - resolve antlr4-python3-runtime dependency clash with multimodal
-    "statsforecast>=1.4.0,<1.5",
+    "statsforecast>=1.7.0,<1.8",
     "mlforecast>=0.10.0,<0.10.1",
-    "utilsforecast>=0.0.10,<0.0.11",
```

Comment:
Can we upper bound this to the currently latest release of `utilsforecast`? We are currently using an older version of `mlforecast` that only gives a lower bound on `utilsforecast`. I'm afraid that something may break in `utilsforecast`, which will break `mlforecast` for us

Reply:
But isn't that something that mlforecast should take care of? We do not have any explicit dependencies on utilsforecast right?